File: DatingApp.SPA/src/app/main/_models/Bank.ts
import { IAccountForLabel } from "./Account";

export interface IBank {
    id : number;
    labelBankShort : string;
    labelBankLong : string;
    addressBank : string;
    postalCodeBank : number;
    adviserFirstName : string;
    adviserLastName : string;
    adviserMail : string;
    adviserFixedPhone : string;
    adviserMobilePhone : string;
    logoClassName : string;
    folderFileSave : string;
}

export interface IBankAccounts {
    id : number;
    labelBankShort : string;
    labelBankLong : string;
    accounts: IAccountForLabel[];
}

File: Budget.MODEL/Dto/_Referential/Bank/Bank.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Budget.MODEL.Dto
{
    public class BankAccountsDto
    {
        public int Id { get; set; }
        public string LabelBankShort { get; set; }
        public string LabelBankLong { get; set; }
        public string LogoClassName { get; set; }
        public List<AccountForDetailDto> Accounts { get; set; }
    }

    public class BankForListDto
    {
        public int Id { get; set; }
        public string LabelBankShort { get; set; }
        public string LabelBankLong { get; set; }
        public string AddressBank { get; set; }
        public int PostalCodeBank { get; set; }
        public string AdviserFirstName { get; set; }
        public string AdviserLastName { get; set; }
        public string AdviserMail { get; set; }
        public string AdviserFixedPhone { get; set; }
        public string AdviserMobilePhone { get; set; }
        public string LogoClassName { get; set; }
        public string FolderFileSave { get; set; }
    }

    //public class BankForSelectDto
    //{
    //    public int Id { get; set; }
    //    public string LabelBankLong { get; set; }
    //}
}
